src/templates/react-native/README.md

```diff
@@ -0,0 +1,87 @@
+# React Native boilerplate
+
+This React Native starter aims to solve commons issues during the development step and build process in the Appcenter. It's a non-opinionated boilerplate, however it uses a couple of preset tools and scripts in order to manage the routine tasks.
+
+## 1. Getting start
+
+**React Native first time**
+
+The doctor command currently supports most of the software and libraries that React Native relies on, such as CocoaPods, Xcode and Android SDK. With doctor we'll find issues with your development environment and give you the option to automatically fix them. If doctor is not able to fix an issue, it will display a message and a helpful link explaining how to fix it manually as the following:
```

Comment:
```suggestion
The doctor command currently supports most of the software and libraries that React Native relies on, such as CocoaPods, Xcode, and Android SDK. With doctor, we'll find issues with your development environment and give you the option to automatically fix them. If doctor is not able to fix an issue, it will display a message and a helpful link explaining how to fix it manually:
```